crate/multiworld/src/lib.rs

```diff
@@ -149,6 +149,7 @@ pub struct RoomFormatter {
     pub password_required: bool,
     pub name: String,
     pub id: u64,
+    pub active: bool,
```

Comment:
Using `Option<RoomFormatter>` where necessary would be better than using this field to create dummy room formatters imo.

Reply:
I tried making this change myself and it turns out to create additional complexity because `Option<RoomFormatter>` doesn't implement `Display`, so the way this is currently implemented is better. I do however think that `active` is not very descriptive so I renamed it to `is_dummy` and inverted it.